packages/matchbox/src/components/UnstyledLink/styles.js

```diff
@@ -0,0 +1,5 @@
+export const text = props => {
+  if (props.as === 'a') {
+    return `text-decoration: underline;`;
```

Comment:
In the last story, this style isn't being applied, but not sure why. Would it make sense to include this in global styles instead of here?

Reply:
<img width="386" alt="Screen Shot 2020-02-28 at 3 52 19 PM" src="https://user-images.githubusercontent.com/3903325/75586409-5d6d3180-5a42-11ea-919d-0ef3bd608b48.png">